frontend/src/components/signup/SectorCard.jsx

```diff
@@ -0,0 +1,49 @@
+import cn from "classnames";
+import { useState } from "react";
+import { SignUpInfo } from "../../context/SignUpContext";
+const SectorCard = ({ sectorInfo }) => {
+  const [isClicked, setIsClicked] = useState(false);
+  const {
+    numberSectorsSelected,
+    setNumberSectorsSelected,
+    setSectorsSelected,
+  } = SignUpInfo();
+
+  if (sectorInfo == null) {
+    return;
+  }
+
+  const handleClick = () => {
+    if (isClicked == true) {
+      setNumberSectorsSelected((self) => self - 1);
```

Comment:
can't you just check the length of `sectorsSelected` array from signupInfo instead? do we need to maintain an extra state if we can just derive it?